extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  const BRL = (price * 0.99 * 5.7);
+ return Math.round((BRL * 100)/100);
```

Comment:
hi 
multiplying by 100 and then dividing by 100 is a really good idea, that eliminates form rounding issues in JavaScript. 
but, you should round, before dividing: 
```js
Math.round(BRL * 100) / 100;
```
otherwise, i gonna get the same result.